### PR TITLE
added prettier config

### PR DESCRIPTION
**Pull request by codeallthethingz**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    trailingComma: "es5",
+    tabWidth: 4,
+    semi: false,
```
Comment (zachbutton) [inline index.js]: We have a strong preference for tab indent. Any opposition to that and `semi: true` ?

**Comment by adarj** [comment]
Personally, I'd prefer semicolons to be enabled.

**Comment by korgon** [comment]
tabs > spaces

I prefer semi-colons as well (at least in Javascript).

**Review by korgon — COMMENTED** [review]
tabs > spaces

I prefer semi-colons as well (at least in Javascript).

**Comment by codeallthethingz** [comment]
@korgon can you mod that file to be how you'd like it?  @zachbutton can you confirm?

**Comment by korgon** [comment]
@codeallthethingz Comitted the changes.

@zachbutton to confirm.

**Review by zachbutton — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (korgon) [reply index.js]: @codeallthethingz
I think this is a good start. Can merge this so that we can begin using it in our projects moving forward?

**Review by zachbutton — APPROVED** [review]

**Comment by zachbutton** [comment]
@codeallthethingz Should we merge this in?

**Pull request merged**